sample-code/spring-app/src/main/java/com/sap/ai/sdk/app/controllers/OrchestrationController.java

```diff
@@ -50,8 +53,10 @@ OrchestrationResponse completion() {
   OrchestrationResponse systemMessage() {
     final var prompt =
         new OrchestrationPrompt(
-            new SystemMessage("Show the user your superior geographical knowledge!"),
-            new UserMessage("What is the capital of France?"));
+            List.of(
+                new SystemMessage("Show the user your superior geographical knowledge!"),
+                new UserMessage("What is the capital of France?")),
+            config);
```

Comment:
(Nitpick)

There goes the powerpoint readiness ;)

Reply:
Maybe moving messages to the second place, so we can use varargs?

OrchestrationPrompt(OrchestrationConfig config, Messages... messages)

Reply:
Fixed 👍🏻